Nav bar

johndelunaaa opened the pull request:
- Updated the Nav to include a pageTitle that is passed from each page
- Imported the nav and set the pageTitle for each of the following pages:
    - HomePage
    - LoginPage
    - RegisterPage

natpdk reviewed (dismissed):
We messaged about it! I made a couple small conflicts with my css coding today

johndelunaaa commented:
@natpdk I corrected the conflicts and removed the page title for the login page and replaced it with my NavTitle setup to pull in the Outdoor Adventures Title with the addition of adding the login page and register page. 

Just tested in my local and it looks like we should be set. Sorry for the inconvenience.

Review and approve when ready :)
